### ISSUE TITLE
[Bug?]: 

### ISSUE THREAD

**Issue by pantheredeye** [redwoodjs/redwood#11402]
### What's not working?

[Related Closed Issue](https://github.com/redwoodjs/redwood/issues/7136#issue-1494962604)

The generator cli tool is not preventing using `/` at the beginning of the <name>.

The generator cli should prevent using `/` at the beginning of the name. It should allow `/` after the first character to create nested pages. 

[from this comment and related issue:](https://github.com/redwoodjs/redwood/issues/7136#issuecomment-1349534277)

> I think we just need a check that the [name] positional argument to these generators can't start with a /. This would be used in any generator that creates a file:
>
> 
> cell
> component
> dataMigration
> directive
> function
> layout
> model
> page
> scaffold
> script
> sdl
> service
> 

---
Here is the current action:

```
$ yarn rw g page /test
← Reverted because: Unexpected token, expected "}" (15:27)
13 |       <p>
14 |         My default route is named <code>c:/programFiles/git/test</code>, link to me with `
15 |         <Link to={routes.c:/programFiles/git/test()}>CProgramFilesGitTest</Link>`
|                           ^
16 |       </p>
17 |     </>
18 |   )
◼ Updating routes file...
◼ Generating types...
◼ One more thing...
```
and

```
$ yarn rw g page /
← Reverted because: Unexpected token, expected "}" (15:27)
13 |       <p>
14 |         My default route is named <code>c:/programFiles/git/</code>, link to me with `
15 |         <Link to={routes.c:/programFiles/git/()}>CProgramFilesGit</Link>`
|                           ^
16 |       </p>
17 |     </>
18 |   )
```

### How do we reproduce the bug?

run `yarn rw g page /`

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by Tobbe** [comment]
@pantheredeye Can you please update the title/name of this issue to be more descriptive? Thanks! 🙏

**Comment by pantheredeye** [comment]
Yep. Funny, I remember typing one out. Maybe I restarted the issue and forgot to write it the second time 🤷‍♂️ 🚀

**Comment by dthyresson** [comment]
@pantheredeye Could you share your Routes.tsx file and show what the route is actually named?
